sdk/src/test/java/com/horizen/transaction/RegularTransactionCreatorTest.java

```diff
@@ -159,9 +161,11 @@ public void beforeEachTest() {
     @Test
     public void RegularTransactionCreator_SuccessCreationTest() {
         List<Pair<PublicKey25519Proposition, Long>> to = new ArrayList<>();
+        List<Pair<MCPublicKeyHash, Long>> withdrawalRequests = new ArrayList<>();
```

Comment:
We have tests for non-empty regular outputs and empty withdrawal. Please, add tests for both specified and only withdrawals specified